test/libethereum/StateTestsFiller/Homestead/stSpecialTestFiller.json

```diff
@@ -266,237 +266,13 @@
         }
     },
 
-    "block504980" : {
```

Comment:
it's a dublicate of 
https://github.com/ethereum/libethereum/blob/develop/test/libethereum/StateTestsFiller/stSpecialTestFiller.json#L269

block 504980 is not of homestead